examples/digitalPlanningApplication/data/landDrainageConsent.ts

```diff
@@ -1,9 +1,9 @@
-import {Schema} from '../../types/Schema';
-import {BaseProposal} from '../../types/schema/data/Proposal';
+import {DigitalPlanningApplication} from '../../../types/schemas/digitalPlanningApplication';
```

Comment:
I'll have a look (in another PR) at path aliases in our `tsconfig.json` - it could be nice to import from `@types`, `@utils` etc.